staramr/detection/AMRDetection.py

```diff
@@ -122,6 +130,32 @@ def run_amr_detection(self, files, pid_threshold, plength_threshold_resfinder, p
         self._detailed_summary_dataframe = self._create_detailed_amr_summary(files, self._resfinder_dataframe,
                                                                              self._pointfinder_dataframe,
                                                                              self._plasmidfinder_dataframe)
+                                                                             
+    def _validate_files(self, files: List[str], ignore_invalid_files: bool) -> List[str]:
+        total_files = len(files)
+        removeable_files = []
+
+        for file in files:
+            # Will raise an error if the input returns an empty generator on non-FASTA files, returns a boolean
+            validInput = any(SeqIO.parse(file, "fasta"))
```

Comment:
Could you add an additional check to make sure `file` is not a directory?

That way, if someone runs with `/dir/files/* --ignore-invalid-files` and there happens to be a directory `/dir/files/dir/` then it will also skip the directory.

Reply:
Resolved this in caf7ed9